xsd/siri_utility/siri_location.xsd

```diff
@@ -202,13 +202,25 @@ Rail transport, Roads and road transport
 		<xsd:annotation>
 			<xsd:documentation>Defines a line shape (since SIRI 2.0)</xsd:documentation>
 		</xsd:annotation>
-		<xsd:sequence>
-			<xsd:element name="Point" type="LocationStructure" minOccurs="2" maxOccurs="unbounded">
+		<xsd:element name="Distance" type="DistanceType" minOccurs="0">
+			<xsd:annotation>
+				<xsd:documentation>Travel distance for the provided shape. In meters.</xsd:documentation>
+			</xsd:annotation>
+		</xsd:element>
+		<xsd:choice>
+			<xsd:sequence>
+				<xsd:element name="Point" type="LocationStructure" minOccurs="2" maxOccurs="unbounded">
```

Comment:
Absolutely not an option. It is either gml:LineString with a gml:posList or sequence of gml:pos not some funky _lets reinvent the wheel, again_.

Reply:
I concur with Stefan.

Reply:
We all agree, but this is the only valid format in the _existing_ datastructure of the `LineShapeStructure`-object (introduced in SIRI 2.0).
(https://github.com/SIRI-CEN/SIRI/blob/integration/xsd/siri_utility/siri_location.xsd#L201-L211)

It is only wrapped in a `<xsd:choice>`-block to preserve backwards-compatibility.

Is your suggestion to remove the backwards compatibility for this element?